grails-app/views/order/_addItemsSummary.gsp

```diff
@@ -1,168 +0,0 @@
-<%@ page import="org.pih.warehouse.order.OrderTypeCode" %>
```

Comment:
Why this file was removed?

Reply:
Yeah that's a bad idea.

Reply:
It's not used anywhere. Everything is done in "order/summary" file.

Reply:
Then I'm fine with that. Are you fine with that too @jmiranda?

Reply:
@DominikaHatala Oh my bad. I saw `_addItems...` and figured it was the template used to render the Add Order Item form. Yeah this looks like it can be removed. Sorry about that. :person_frowning: